styles.css

```diff
@@ -0,0 +1,246 @@
+
+    --featured: #90949F;
+    --featured-price: #1C202E;
+    --featured-shadow: rgba(255, 255, 255, 0.1);
+
+    --white: #FCFCFC;
+}
+
+body {
+    font-family: sans-serif;
+    /* background-color: brown; */
+    margin: 0;
+    padding: 0;
+    background-image: 
+    linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), 
+    url('./media/background.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: fixed;
+    background-position: center;
+  }
+
+.main-container {
+    display: flex;
+    flex-direction: column;
+    gap: 60px;
+    /* background-color: blue; */
+    /* padding: 25px; */
+    padding: 70px 0 70px 0;
+    width: 80%;
+    text-align: center;
+    color: var(--white);
+    font-weight: 400;
+    margin: auto;
+
+}
+
+.titles{
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+    padding: 0;
+    gap: 30px;
+}
+
+h1 {
+    font-size: 36px;
+    margin: 0;
+    padding: 0;
+    font-family: 'Ubuntu', sans-serif;
+    font-weight: 500;
+    /* margin-top: 20px; */
+    /* background-color: blueviolet; */
+  }
+
+h3 {
+    font-size: 22px;
+    margin: auto;
+    padding: 0;
+    font-family: 'Manrope', sans-serif;
+    font-weight: 400;
+    width: 75%;
+    max-width: 540px;
+    text-align: center;
+    /* background-color: cornsilk; */
+  }
+
+.pricing-grid {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, 255px);
+    grid-gap: 30px;
+    justify-content: center;
+    font-family: 'Manrope', sans-serif;
+    /* padding-top: 30px;
+    padding-bottom: 75px; */
+}
+
+h2 {
+    font-weight: 400;
+    font-size: 18px;
+}
+
+hr {
+    border-top: 1px solid #D1D2D9;
+    width: 100%;
+    gap: 0px;
+    padding: 0;
+    margin: 0;
+}
+
+.pricing-grid p {
+    size: 14px;    
+}
+
+.price-tag {
+    font-size: 22px;
+}
+
+.pricing-grid button {
+    font-family: 'Manrope', sans-serif;
+    width: 115px;
+    height: 38px;
+    border-radius: 50px;
+    margin-bottom: 15px;
+    margin: 15px 0 15px 0;
+    border: 0;
+}
+
+.pricing-grid button:hover {
+    transform: scale(1.05);
+    cursor: pointer;
+}
+  
+.pack {
+    /* background-color: chartreuse; */
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 0px;
+    padding: 15px;
+    border: 1px solid var(--white);
+    border-radius: 3px;
+    transition: background-color 0.3s ease;
+}
+
+.pack:hover {
+    transform: scale(1.02);
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.07), 0 2px 4px rgba(0, 0, 0, 0.03);
+    background-color: rgba(255, 255, 255, 0.2);}
+
+.circle-icon {
+    width: 80px;
+    height: 80px;
+    background-color: var(--white);
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+.icon {
+    width: 29.87px;
+    height: 29.87px;
+  }
+
+
+
+/* Plan specific colors */
+.bravo {
+    box-shadow: 0px 9px 30px 0px var(--bravo-circle-shadow);
+}
+.bravo-btn {
+    color: var(--bravo);
+}
```

Comment:
I can see what you where doing here, but there is no need to create a class called "bravo-btn", insead what you can do its apply the bravo class to the father div, and with that you can do something like:

```
.bravo button{
    color: var(--bravo);
}
```

But its okey! there is no need to change it, its just a suggestion for the future!

Reply:
Thanks!! This is really helpfull to shape the way I think when creating classes!! will implement